source/manual/datalabs-setup-local-machine.html.md

```diff
@@ -0,0 +1,42 @@
+---
+owner_slack: "#govuk-data-labs"
+title: Set up your local machine
+section: Data Labs Team Manual
+layout: manual_layout
+parent: "/manual.html"
+---
+
+When you join the GOV.UK Data Labs team, you must set up your local machine and join Google groups.
+
+## Set up your local machine
+
+To set up your local machine, follow the [Get started on GOV.UK documentation](https://docs.publishing.service.gov.uk/manual/get-started.html).
+
+Once you have set up your local machine by following this documentation, you should have access to GitHub and AWS. If you are a data scientist, ask in the [GOV.UK Data Labs slack channel](https://gds.slack.com/archives/CHR4UQKU4) for someone to add you to the:
+
+- `ukgovdatascience` organisation in GitHub
```

Comment:
- `alphagov` organisation in Github too

Reply:
ukgovdatascience is the GDS DS community github org, alphagov is GOV.UK github org more generally

Reply:
if you've followed the get started process linked further up, you will already be a member of the alphagov organisation

Reply:
ah, cool, good point